packages/shared-influx-pieces/src/lib/AnalyticsListener.ts

```diff
@@ -0,0 +1,48 @@
+import type { Point } from '@influxdata/influxdb-client';
+import { envParseBoolean, envParseString } from '@skyra/env-utilities';
+import { Listener, type PieceContext } from '@skyra/http-framework';
+import { Tags } from './AnalyticsSchema.js';
+
+export abstract class AnalyticsListener extends Listener {
+	public tags: [Tags, string][] = [];
+
+	public constructor(context: PieceContext, options: AnalyticsListener.Options) {
+		super(context, { ...options, enabled: envParseBoolean('INFLUX_ENABLED') });
```

Comment:
It should use `isInfluxInitialized()` instead. And be reversed so `enabled` can be overridden.